src/views/profile/bookmarks-view/helpers/card-sort-data.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiBookmark } from 'lib/learn-client/api/api-types'
+
+export const SortData = {
+	newest: {
+		text: 'Newest',
+		sort: (a: ApiBookmark, b: ApiBookmark) => {
+			return new Date(b.created_at).getTime() - new Date(a.created_at).getTime()
+		},
+	},
+	oldest: {
+		text: 'Oldest',
+		sort: (a: ApiBookmark, b: ApiBookmark) => {
+			return new Date(a.created_at).getTime() - new Date(b.created_at).getTime()
+		},
+	},
+}
```

Comment:
I really like this structure! It came about with the bookmark button as well. Very clear and consumable.